src/constants.ts

```diff
@@ -29,7 +29,7 @@ export const MEMBER_NOT_FOUND_CODE = "member.notFound";
 export const MEMBER_NOT_FOUND_MESSAGE = "member.notFound";
 export const MEMBER_NOT_FOUND_PARAM = "member";
 
-export const MEMBERSHIP_REQUEST_NOT_FOUND = "Membership Request not found";
+export const MEMBERSHIP_REQUEST_NOT_FOUND = "MembershipRequest already exists";
```

Comment:
Why this text is changed?

Reply:
I thought by adding extra constant just  for the single message "MembershipRequest already exists" will be affecting the test coverage.

Reply:
Should I create one constant as 
`export const MEMBERSHIP_REQUEST_ALREADY_EXISTS = "MembershipRequest already exists";` then use it in `[src/resolvers/Mutation/sendMembershipRequest.ts]`

Reply:
@Manoramsharma Why would name the variable that stores the message `MembershipRequest already exists` as `MEMBERSHIP_REQUEST_NOT_FOUND`? Does it make sense?

Reply:
> @Manoramsharma Why would name the variable that stores the message `MembershipRequest already exists` as `MEMBERSHIP_REQUEST_NOT_FOUND`? Does it make sense?

Yeah, I will be adding a separate variable MEMBERSHIP_REQUEST_ALREADY_EXISTS that stores that value.